dnf-behave-tests/features/updateinfo.feature

```diff
@@ -312,3 +312,32 @@ Examples:
     # yum compatibility
     | list bugzillas      |
     | list bzs            |
+
+
+@bz1728004
+Scenario: updateinfo show <advisory> of the running kernel after a kernel update
+   When I execute dnf with args "install kernel"
+   Then the exit code is 0
+    And Transaction is following
+        | Action        | Package                                  |
+        | install       | kernel-core-0:4.18.16-300.fc29.x86_64    |
+        | install       | kernel-modules-0:4.18.16-300.fc29.x86_64 |
+        | install       | kernel-0:4.18.16-300.fc29.x86_64         |
+  Given I use repository "dnf-ci-fedora-updates"
+    And I execute dnf with args "updateinfo list kernel"
+   Then the exit code is 0
+    And stdout matches line by line
+    """
+    <REPOSYNC>
+    FEDORA-2019-348e185000 bugfix kernel-4.19.15-300.fc29.x86_64
+    """
+   When I execute dnf with args "update kernel"
+   Then the exit code is 0
+    And Transaction is following
+        | Action        | Package                                  |
+        | install       | kernel-core-0:4.19.15-300.fc29.x86_64    |
+        | install       | kernel-modules-0:4.19.15-300.fc29.x86_64 |
+        | install       | kernel-0:4.19.15-300.fc29.x86_64         |
+   When I execute dnf with args "updateinfo list kernel"
+   Then the exit code is 0
+    And stdout does not contain "FEDORA-2019-348e185000"
```

Comment:
You should also use `stdout is` or a more loose positive match test, the negative match is quite fragile...

Reply:
I agree but unfortunately it's the only match possible

Reply:
How is that?

(if you say something's not possible, you'll save a communication round trip if you include the reasoning straight away :wink: )

Reply:
This code follow the AC because checks that the previous kernel advisory is not present

Reply:
Code corrected